examples/templates/typescript-template/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```

Comment:
Maybe we can use `@tsconfig/create-react-app` here, so we don't have to mess around with the config ourselves.

https://www.npmjs.com/package/@tsconfig/create-react-app

But as this is just a copy, feel free to ignore this comment